semantic-python/semantic-python.cabal

```diff
@@ -0,0 +1,43 @@
+cabal-version:       2.4
+
+name:                semantic-python
+version:             0.0.0.0
+synopsis:            Semantic support for Python.
+description:         Semantic support for Python using the semantic-core intermediate language.
+homepage:            https://github.com/github/semantic/tree/master/semantic-python#readme
+bug-reports:         https://github.com/github/semantic/issues
+license:             MIT
+license-file:        LICENSE
+author:              The Semantic authors
+maintainer:          opensource+semantic@github.com
+copyright:           (c) 2019 GitHub, Inc.
+category:            Language
+build-type:          Simple
+stability:           alpha
+extra-source-files:  README.md
+
+tested-with:         GHC == 8.6.4
+
+library
+  exposed-modules:
+    Language.Python.Core
+  -- other-modules:
+  -- other-extensions:
+  build-depends:       base ^>=4.12.0.0
+                     , semantic-core ^>= 0.0.0.0
+                     , tree-sitter ^>= 0.1.0.0
+                     , tree-sitter-python ^>= 0.1.0.1
```

Comment:
We’ll need to bump these when we push new releases of them following https://github.com/tree-sitter/haskell-tree-sitter/pull/144 being merged.

In the meantime, `semantic-python` won’t build unless you edit `cabal.project` locally (or maybe `cabal.project.local`) to point it at https://github.com/tree-sitter/haskell-tree-sitter/pull/144.